diff --git a/opengever/core/testing.py b/opengever/core/testing.py
index e225868..2a0f7ae 100644
--- a/opengever/core/testing.py
+++ b/opengever/core/testing.py
@@ -196,6 +196,7 @@ class OpengeverFixture(PloneSandboxLayer):
     def setUpPloneSite(self, portal):
         self.installOpengeverProfiles(portal)
         self.setupLanguageTool(portal)
+        self.allowAllTypes(portal)
         deactivate_activity_center()
         deactivate_bumblebee_feature()
 
@@ -212,45 +213,7 @@ class OpengeverFixture(PloneSandboxLayer):
         os.environ['BUMBLEBEE_DEACTIVATE'] = "True"
 
     def installOpengeverProfiles(self, portal):
-        # Copied from metadata.zxml of opengever.policy.base:default
-        # The aim is to use the opengever.policy.base:default here, but it
-        # changes some things such as the language which will result in
-        # lots of failing tests.
-        applyProfile(portal, 'plone.app.dexterity:default')
-        applyProfile(portal, 'plone.app.registry:default')
-        applyProfile(portal, 'plone.app.relationfield:default')
-        applyProfile(portal, 'opengever.globalindex:default')
-        applyProfile(portal, 'opengever.ogds.base:default')
-        applyProfile(portal, 'opengever.base:default')
-        applyProfile(portal, 'opengever.document:default')
-        applyProfile(portal, 'opengever.mail:default')
-        applyProfile(portal, 'opengever.dossier:default')
-        applyProfile(portal, 'opengever.repository:default')
-        applyProfile(portal, 'opengever.journal:default')
-        applyProfile(portal, 'opengever.task:default')
-        applyProfile(portal, 'opengever.tabbedview:default')
-        applyProfile(portal, 'opengever.trash:default')
-        applyProfile(portal, 'opengever.inbox:default')
-        applyProfile(portal, 'opengever.tasktemplates:default')
-        applyProfile(portal, 'opengever.portlets.tree:default')
-        applyProfile(portal, 'opengever.contact:default')
-        applyProfile(portal, 'opengever.advancedsearch:default')
-        applyProfile(portal, 'opengever.sharing:default')
-        applyProfile(portal, 'opengever.latex:default')
-        applyProfile(portal, 'opengever.meeting:default')
-        applyProfile(portal, 'opengever.activity:default')
-        applyProfile(portal, 'opengever.bumblebee:default')
-        applyProfile(portal, 'opengever.officeatwork:default')
-        applyProfile(portal, 'opengever.officeconnector:default')
-        applyProfile(portal, 'opengever.private:default')
-        applyProfile(portal, 'ftw.datepicker:default')
-        applyProfile(portal, 'plone.formwidget.autocomplete:default')
-        applyProfile(portal, 'plone.formwidget.contenttree:default')
-        applyProfile(portal, 'ftw.contentmenu:default')
-        applyProfile(portal, 'ftw.zipexport:default')
-        applyProfile(portal, 'opengever.disposition:default')
-        applyProfile(portal, 'plone.restapi:default')
-
+        applyProfile(portal, 'opengever.core:default')
         applyProfile(portal, 'opengever.testing:testing')
 
     def createMemberFolder(self, portal):
@@ -263,23 +226,22 @@ class OpengeverFixture(PloneSandboxLayer):
     def setupLanguageTool(self, portal):
         """Configure the language tool as close as possible to production,
         without breaking most of the existing tests.
-
-        For production, the language tool is configured in
-        opengever.policy.base:default, which we don't import here
-        (see comment in installOpengeverProfiles() above).
         """
         lang_tool = api.portal.get_tool('portal_languages')
-        lang_tool.use_combined_language_codes = True
-        lang_tool.display_flags = False
-        lang_tool.start_neutral = False
-        lang_tool.use_subdomain_negotiation = False
-        lang_tool.authenticated_users_only = False
-        lang_tool.use_request_negotiation = True
-
-        # These would be (possible) production defaults, but will break tests
-        # lang_tool.setDefaultLanguage('de-ch')
-        # lang_tool.supported_langs = ['fr-ch', 'de-ch']
+        lang_tool.setDefaultLanguage('en')
+        lang_tool.supported_langs = ['en']
 
+    def allowAllTypes(self, portal):
+        """We used to have filter_content_types=False for the Plone Site
+        while running tests.
+        With the profile merge, this has changed.
+        For keeping the change simple, we are allowing all types, making
+        the tests run again.
+        However, this should be cleaned up at some point by updating all
+        tests which rely on that.
+        """
+        portal.portal_types['Plone Site'].filter_content_types = False
+
 
 class MemoryDBLayer(Layer):
     """A Layer which only set up a test sqlite db in to the memory
