diff --git a/opengever/core/testing.py b/opengever/core/testing.py
index d4ffe76..05cf5f8 100644
--- a/opengever/core/testing.py
+++ b/opengever/core/testing.py
@@ -213,7 +213,7 @@ class OpengeverFixture(PloneSandboxLayer):
         os.environ['BUMBLEBEE_DEACTIVATE'] = "True"
 
     def installOpengeverProfiles(self, portal):
-        applyProfile(portal, 'opengever.policy.base:default')
+        applyProfile(portal, 'opengever.core:default')
         applyProfile(portal, 'opengever.testing:testing')
 
     def createMemberFolder(self, portal):
