diff --git a/opengever/dossier/tests/test_templatedossier.py b/opengever/dossier/tests/test_templatedossier.py
index 99232fd..64de2a1 100644
--- a/opengever/dossier/tests/test_templatedossier.py
+++ b/opengever/dossier/tests/test_templatedossier.py
@@ -606,7 +606,7 @@ class TestTemplateDossierListings(FunctionalTestCase):
     def test_enabled_actions_are_limited_in_document_tab(self, browser):
         browser.login().open(self.templatedossier, view=DOCUMENT_TAB)
 
-        self.assertEqual(
+        self.assertItemsEqual(
             ['Copy Items', 'Checkin with comment', 'Checkin without comment',
              'Export selection', 'trashed', 'Export as Zip'],
             browser.css('.actionMenuContent li').text)
@@ -627,7 +627,7 @@ class TestTemplateDossierListings(FunctionalTestCase):
     def test_enabled_actions_are_limited_in_sablontemplates_tab(self, browser):
         browser.login().open(self.templatedossier, view=SABLONTEMPLATES_TAB)
 
-        self.assertEqual(
+        self.assertItemsEqual(
             ['Copy Items', 'Checkin with comment', 'Checkin without comment',
              'Export selection', 'trashed', 'Export as Zip'],
             browser.css('.actionMenuContent li').text)
