diff --git a/opengever/repository/tests/test_deletion.py b/opengever/repository/tests/test_deletion.py
index 5988d0e..f8d000f 100644
--- a/opengever/repository/tests/test_deletion.py
+++ b/opengever/repository/tests/test_deletion.py
@@ -69,8 +69,7 @@ class TestRepositoryDeletion(FunctionalTestCase):
         create(Builder('dossier').within(self.repository))
         browser.login().open(self.repository)
         self.assertEquals(
-            ['Export as Zip',
-             'Prefix Manager',
+            ['Prefix Manager',
              'Properties',
              'Sharing',
              'repositoryfolder-transition-inactivate'],
@@ -81,7 +80,6 @@ class TestRepositoryDeletion(FunctionalTestCase):
         browser.login().open(self.repository)
         self.assertEquals(
             ['Delete',
-             'Export as Zip',
              'Prefix Manager',
              'Properties',
              'Sharing',
