diff --git a/opengever/private/tests/test_dossier.py b/opengever/private/tests/test_dossier.py
index af51dcd..cb4c444 100644
--- a/opengever/private/tests/test_dossier.py
+++ b/opengever/private/tests/test_dossier.py
@@ -81,13 +81,13 @@ class TestPrivateDossier(FunctionalTestCase):
                             .within(dossier)
                             .having(responsible=TEST_USER_ID))
 
-        self.assertSequenceEqual(
+        self.assertItemsEqual(
             ['opengever.document.document',
              'ftw.mail.mail',
              'opengever.private.dossier'],
             [fti.id for fti in dossier.allowedContentTypes()])
 
-        self.assertSequenceEqual(
+        self.assertItemsEqual(
             ['opengever.document.document', 'ftw.mail.mail'],
             [fti.id for fti in subdossier.allowedContentTypes()])
 
